### ISSUE TITLE
[feature request] default value for the app-name

### ISSUE THREAD

**Issue by masonlouchart** [convoo/login-fire#152]
100% of the applications using the `login-fire` element, uses it only once (I really hope... 🙏 ). I don't know for the `firebase-app` element but I don't think the majority of applications uses multiple Firebase configurations.

So, why do not set the `appName` property to the same value of the property `name` of the first `firebase-app` element found in the document:question:

**Comment by Oupsla** [comment]
Personally, I'm using multiple firebase-app elements to switch easily between my pre-prod and prod environment
So I prefer to choose appName

**Comment by masonlouchart** [comment]
A default value for `appName` will not prevent you to set the value if you want to.

IMHO, the kind of stuff like define the environment to use by your app must be done at the compile step. For instance, I use [preprocessing](https://github.com/jsoverson/preprocess#directive-syntax) to replace my Firebase configuration when the app is compiled. Doing so, the app served to our customers has only one configuration and I can use the exact same source code on different Firebase apps running one command line (`npm run deploy[:dev|prod]`).

I try to avoid conditionnal statements/executions as uch as possible. The API REST URL the app has to use, the API keys, the target Firebase app, logs level, app title, app icon, all of this kind of things is done at the compile step.

Maybe you're more familiar with Maven's profiles.

**Comment by Oupsla** [comment]
:open_mouth: I'll try preprocessing right now ! Thank you so much for the tip !  

So it makes sense yes, but it is safe trying to retrieve the firebase name from the login-fire element ?  
I'm telling this cause all polymerfire elements don't try to do this, you have to manually set a appName if you set a name to your firebase-app.

**Comment by masonlouchart** [comment]
You're right, Firebase elements do not read this property from the `firebase-app` 🤔 . This is why I prefered ask about this "feature".

In fact, this is not a big improvement by itself. This topic is related to the question about introduce a new `login-fire-splash-screen` element (#151) which I wouldn't like have set an appName to.
